### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Mulan-94**
- Fixing #49 
- Attempting at #50 
- Implementing some changes in #51
- Changed time axis to UTC time
- Display all times in bandpass plots
- Add download selected data button
- Adds averaging feature in `ragavi-vis`
- Move statistics to table on plot.

### CODE REVIEW

File: ragavi/utils.py [hunk 1]
```diff
@@ -544,8 +559,11 @@ def __handle_uncaught_exceptions(extype, exval, extraceback):
 
 ########################################################################
 ########################### Some useful functions ######################
+
 def __welcome():
     """Welcome to ragavi"""
+    from pyfiglet import Figlet
```
Comment (Athanaseus) [inline ragavi/utils.py]: Any reason for this import.

Comment (Mulan-94) [reply ragavi/utils.py]: It's not exactly in use.

**Comment by Athanaseus** [comment]
Since `ragavi` is on full py3 support. Perhaps you want to update the shebang line in the binary file.
Also, add `python_requires='>=3.6'` or whatever minimum py3 that is supported in the `setup.py`.

Furthermore, it also seems that you added some new functionality, (i.e. averaging feature &  download selected data) and improvements (statistics to table), this will require a `MINOR` version increment instead of a `PATCH`.

File: ragavi/utils.py (hunk 1 above)
Comment (Athanaseus) [reply ragavi/utils.py]: Then I guess this should be removed because i see it's not even in the requirements.

**Comment by Mulan-94** [comment]
Don't merge yet, I discovered some error in function. Working towards fixing it.

**Comment by Mulan-94** [comment]
Ok, now ready to merge and release.

File: ragavi/utils.py (hunk 1 above)
Comment (Mulan-94) [reply ragavi/utils.py]: I've commented it out now. Can I merge?

Comment (Athanaseus) [reply ragavi/utils.py]: Yeah go ahead :+1:

**Pull request merged**